datafusion/core/tests/sqllogictests/test_files/information_schema.slt

```diff
@@ -103,9 +103,10 @@ query R
 SHOW ALL
 ----
 datafusion.catalog.create_default_catalog_and_schema true
+datafusion.catalog.format NULL
+datafusion.catalog.has_header false
 datafusion.catalog.information_schema true
 datafusion.catalog.location NULL
-datafusion.catalog.type NULL
```

Comment:
`type` is a reserved keyword in rust, and the name was pretty overloaded anyway, so I opted to rename this to `format`. Amusingly this was already what the local variables were named as